docs/migrating/from-gatsby.md

```diff
@@ -189,16 +189,20 @@ Instead, use the built-in [`next/image`](/docs/api-reference/next/image.md) comp
 
 ```jsx
 import Image from 'next/image'
+import profilePic from '../public/me.png'
 
 export default function Home() {
   return (
     <>
       <h1>My Homepage</h1>
       <Image
-        src="/me.png"
+        src={profilePic}
         alt="Picture of the author"
-        width={500}
-        height={500}
+        // width={500} automatically provided
+        // height={500} automatically provided
+        // blurDataURL="data:..." automatically provided
+        // Optionally allows to add a blurred version of the image while loading
+        // placeholder="blur"
```

Comment:
Lets show the gatsby equivalent props here so users can more easily migrate

For example `layout` would be important. I think gatsby `fixed` is the same as ours but `fluid` is probably `responsive` for next.

Also we should comment with the "blur up" placeholder equivalent for gatsby

Reply:
```suggestion
        // When "responsive", similar to "fluid" from Gatsby
        // When "intrinsic", similar to "fluid" with maxWidth from Gatsby
        // When "fixed", similar to "fixed from "Gatsby"
        layout="responsive"
        // Optional, similar to "blur-up" from Gatsby
        placeholder="blur"
        // Optional, similar to width in graphql
        width={500}
        // Optional, similar to height in graphql
        height={500}
```